### PR TITLE
Kustomize deployment files

### PR DESCRIPTION
**Pull request by dvaldivia**

### CODE REVIEW

File: k8s/base/minio-operator.yaml [hunk 1]
```diff
@@ -203,86 +202,9 @@ spec:
       serviceAccountName: minio-operator-sa
       containers:
         - name: minio-operator
-          image: minio/k8s-operator:2.0.2
+          image: minio/k8s-operator:2.0.4
```
Comment (kerneltime) [inline k8s/base/minio-operator.yaml]: Needs to be 2.0.5

File: README.md [hunk 2]
```diff
@@ -3,72 +3,17 @@ MinIO Kubernetes Cloud
 
 ## Installation
```
Comment (kerneltime) [inline README.md:4]: Add a line for make yamlgen?

File: k8s/base/minio-operator.yaml (hunk 1 above)
Comment (dvaldivia) [reply k8s/base/minio-operator.yaml]: done

File: README.md (hunk 2 above)
Comment (dvaldivia) [reply README.md:4]: that's only for when we want a new yaml for operator to be checked in

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by kerneltime — APPROVED** [review]

**Pull request merged**